nodejs-ipaddr.js/nodejs-ipaddr.js.spec

```diff
@@ -0,0 +1,56 @@
+# FIXME
+# Remove nodejs_symlink_deps if installing bundled module with NPM
+
+%global npm_name ipaddr.js
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 1.2.0
+Release: 1%{?dist}
+Summary: A library for manipulating IPv4 and IPv6 addresses in JavaScript
+License: MIT
+URL: FIXME
+Source0: http://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
+ExclusiveArch: %{nodejs_arches} noarch
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
+%{?nodejs_find_provides_and_requires}
+
+%description
+%{summary}
+
```

Comment:
Missing this, which adds the ipaddr.js-doc subpackage:

```
%package doc
Summary: Documentation for nodejs-%{npm_name}
Group: Documentation
Requires: nodejs-%{npm_name} = %{version}-%{release}
BuildArch: noarch

%description doc
This package contains documentation for nodejs-%{npm_name}
```

The template should have this too, I removed it a while ago from the default template when I merged a PR a person from the packaging team in Fedora but we do use it.